site/homepage/content/release-notes.md

```diff
@@ -15,6 +15,7 @@ description = "Information about changes in recent Hono releases. Includes new f
   The timeout is configured with the property `idleTimeout`. This determines if a connection will timeout and be closed
   if no data is received or sent within the idle timeout period. The idle timeout is in seconds.
   A zero value means no timeout is used.
+* The MQTT adapter skips command or error (the first one) subscription if both are requested for the same device.
```

Comment:
FMPOV, it doesn't directly become clear whether this is a fix or an enhancement.
How about something like:
```
The MQTT adapter skipped (the first one of) a command or error subscription if both were requested for the same device. This has been fixed.
```